src/test/java/hw1/CalculatorDivideTest.java

```diff
@@ -0,0 +1,66 @@
+package hw1;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class CalculatorDivideTest {
+
+    private Calculator calculator;
+
+    @BeforeMethod
+    public void setCalculator() {
+        calculator = new Calculator();
+    }
```

Comment:
I could see in the following classes code duplication: 
- CalculatorDivideTest
- CalculatorMultiplyTest
- CalculatorSubtractTest
- CalculatorSumTest